website/models.py

```
@@ -640,6 +642,43 @@ def get_cve_score(self):
+
+            owner = parts[3]
+            repo = parts[4]
+            issue_number = parts[6]
+
+            api_url = f"https://api.github.com/repos/{owner}/{repo}/issues/{issue_number}"
+
+            headers = {"Accept": "application/vnd.github.v3+json"}
+
+            from django.conf import settings
+
+            if hasattr(settings, "GITHUB_TOKEN") and settings.GITHUB_TOKEN != "blank":
```

Ruff (0.14.5)

Possible hardcoded password assigned to: "GITHUB_TOKEN"

(S105)